### ISSUE TITLE
[BUG] Advancement doesn't have a name

### ISSUE THREAD

**Issue by FewerFlaws** [DaFuqs/StarrySky#7]
**Describe the bug**
When landing on a nether wart tree sphere in the starry sky nether I got an achievement that showed up with a namespace name.  There is likely a typo in the advancements json.

**To Reproduce**
Enter the starry sky nether dimension, find a crimson forest sphere that looks like the image below and land on it.

**Expected behavior**
Advancement pop-ups are in plain english.

**Screenshots**
![2021-01-09_16 15 24](https://user-images.githubusercontent.com/75643699/104111391-15d84880-5296-11eb-9732-4b860e5d199d.png)

**Comment by FewerFlaws** [comment]
Oh, crimson wood sphere:
```
[16:12:09] [Server thread/INFO]: Josheva has made the advancement [advancements.dafuqs.starry_sky.sphere_crimson_wood.title]
[16:12:09] [main/INFO]: [CHAT] Josheva has made the advancement [advancements.dafuqs.starry_sky.sphere_crimson_wood.title]
```

**Comment by DaFuqs** [comment]
Ooops. There was a single dot missing with that one. Fixed, thank you!
